src/lib/components/primitives/button/button.story.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+  import Button from "./button.svelte";
+
+  // Button props
+  export let busy = false;
+  export let onClick = (event: Event) => {};
+  export let style: 'outline' | 'primary' | 'secondary' = 'outline';
+  export let text: string;
+  export let type: 'button' | 'submit' = null;
+  export let width: 'auto' | 'full' | 'half' = 'auto';
+</script>
+
+<Button {busy} {onClick} {style} {type} {width}>
```

Comment:
Is there a reason we are not using https://storybook.js.org/docs/svelte/essentials/controls would be good to use here I think for this component? It may make the story a bit clean

Reply:
We need to discuss this as it's a bit complicated, but `*.story.svelte` is the Svelte component that's used by `*.stories.ts`. The `*.stories.ts` file is where you'll find the "controls" aspect to the story, not the component.

I've separate the component from the official `*.stories.ts` file because of this: https://github.com/storybookjs/storybook/issues/16673#issuecomment-1133477439. So, I'm avoiding using Svelte components as the official Storybook "stories" file, so we can have full functionality of Storybook. If we use the Svelte CSF (`@storybook/addon-svelte-csf`), we can't use `storyStorev7` and a bunch of other stuff, like interaction testing.